Changes requested by Ivan (#55)
SW814F20/Report: report/Process/ImpactOnOthers.tex

Where most software project being conducted using agile development methods require access to an onsite customer or Product Owner, the current rules of social distancing and the current shutdown of whole counties, the accessability to such an onsite customer will be limited.
Since the Product Owner is not a developer, they should not be expected to learn and use GitHub to communicate with the developers them self and thus a tool like the product of this project would come in handy.

While others may not use the same development process and workflow as we have, the tools we have utilized is still very applicable and may come in handy for others.
While different groups/teams, that being students or employees, will often have different preferences even within the same faculty/company, each group/team will have to find the tools that enable them to perform the most.

> During future pandemics, the world will have fresh in its memory how to deal with the situation, as it has happened before.

Reviewer: ```suggestion
During future pandemics, the world will have a better idea of how to deal with the situation, as it has happened before.
```
